README.md

```diff
@@ -10,7 +10,43 @@ by adding `secrex` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:secrex, github: "forzafootball/secrex", tag: "v0.1.0"}
+    {:secrex, github: "ForzaElixir/secrex", tag: "v0.1.0", runtime: false}
   ]
 end
 ```
+
+## Usage
+
+Secrex requires some configuration in order to work. For example, in config/config.exs:
+
+```elixir
+config :secrex,
+  key_file: ".secrets_key",
+  files: ["config/env/prod.secret.exs"]
+```
+
+* `key_file` is a path for a file that contain a key that will be used for encrypting and decrypting
```

Comment:
a path _to_ the key file that will be used for encryption and decryption?

Reply:
perhaps _if this is not configured, you will be prompted to enter later_?

Reply:
dun